README.md

```diff
@@ -60,5 +60,70 @@ To register a device, run as root from LmP device:
+
+After successfully executing the fake-lmp-device-register, a new directory, `var/sota`, will be created in the current 
+directory. This contains a curl script and the necessary certificates.
+
+Navigate to this directory:
+
+```shell
+cd var/sota
+```
+
+Inspect the server URL configured in the `sota.toml` file:
+
+```shell
+$ cat sota.toml 
+```
+
+You should see an output similar to:
+
+```shell
+[tls]
+server = "https://ec056c71-d698-41a6-bea0-1f1ac8c44bde.ota-lite.foundries.io:8443"
+ca_source = "file"
+pkey_source = "file"
+cert_source = "file"
+...
+```
+
+Now, test the connectivity with the server using curl:
+
+```shell
+$ ./curl https://ec056c71-d698-41a6-bea0-1f1ac8c44bde.ota-lite.foundries.io:8443/repo/root.json                                                                 
+```
+
+You should see an output similar to:
+
+```shell
+{"signatures":[{"keyid":"69d9536e4c859423f7125fbff3ec73e714dcd04af2cdb144874ad470d5be0f35","method":"ed25519","sig":"a3PIu6wI41lCp24i+RzIX93S5WyBgFhs+NjEu7OXGy9fdKpgEZCoFzNBNrfCUCBTKSY1Cf3VDkhnNx8j43SCAQ=="}],"signed":{"_type":"Root","consistent_snapshot":false,"expires":"2024-10-31T12:00:04Z","keys":{"4bab04725e4baf17798638f905008281eaa7ad39392b42085cf75fb8cd2f7545":{"keytype":"ED25519","keyval":{"public":"4526865f7fb1ab13134f55c9e18cb95a102982d459cb2016484a1ab5d5f0bf56"}},"69d9536e4c859423f7125fbff3ec73e714dcd04af2cdb144874ad470d5be0f35":{"keytype":"ED25519","keyval":{"public":"be1110c45005ef4a4b36befb1f2983f796d8639217670a18ea5954a754ae6331"}},"7f670abf569887cbe43fdfd40b85534926a68a21e922b1f87be6bc7075cb26cc":{"keytype":"ED25519","keyval":{"public":"5f69edd4cfb0b25c60511d3a3c4b9256d599012d4afbfaba76af70ca5c3aa290"}},"a8971465c5bae6e6e76a2396f1d8a5d1ef0df3ec54b92586a6cc645cdda73fc3":{"keytype":"ED25519","keyval":{"public":"62ed3aedb49c1798f89d4d54d507da1db0442383afc9a6a5cde942186b54825a"}},"fdc1c49613628e6426848cfe49697775adef80600174b2de5499e9cbb0b07010":{"keytype":"ED25519","keyval":{"public":"a01364d8a74e56bface1676b8b22bdc1559b4f5c6e544eecf80a4817672acc29"}}},"roles":{"root":{"keyids":["69d9536e4c859423f7125fbff3ec73e714dcd04af2cdb144874ad470d5be0f35"],"threshold":1},"snapshot":{"keyids":["4bab04725e4baf17798638f905008281eaa7ad39392b42085cf75fb8cd2f7545"],"threshold":1},"targets":{"keyids":["7f670abf569887cbe43fdfd40b85534926a68a21e922b1f87be6bc7075cb26cc","fdc1c49613628e6426848cfe49697775adef80600174b2de5499e9cbb0b07010"],"threshold":1},"timestamp":{"keyids":["a8971465c5bae6e6e76a2396f1d8a5d1ef0df3ec54b92586a6cc645cdda73fc3"],"threshold":1}},"version":4,"x-changelog":{"message":"Rotate TUF targets offline signing key","polis-id":"65396079189e51946004fc5c","timestamp":"2023-10-31T12:01:28Z"},"x-key-owners":{"69d9536e4c859423f7125fbff3ec73e714dcd04af2cdb144874ad470d5be0f35":{"created-at":"2023-10-31T12:00:04Z","polis-id":"65396079189e51946004fc5c"},"fdc1c49613628e6426848cfe49697775adef80600174b2de5499e9cbb0b07010":{"created-at":"2023-10-31T12:01:28Z","polis-id":"65396079189e51946004fc5c"}}}}%    
```

Comment:
Again, this is too much for the README. FWIW 60% of this content will be different for the user.

Reply:
Maybe we shouldn't even highlight this tool. Couple of notes about this:
 * I'm not aware of anyone using it (certainly not to the extent we devote much energy on it)
 * nobody can trust it because its *not* lmp-device-register

We should probably keep a user's focus on running the server itself and not this. Maybe we put the script under `contrib/` to make it more clear its not a primary piece of tooling.